css.css

```diff
@@ -0,0 +1,68 @@
+body {
+    font-family: 'Baloo 2', cursive;
+	margin: 5em;
+}
+
+.collectionContainer1{
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+}
+
+.collectionContainer2{
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+}
```

Comment:
This might be redundant - consider making these two the same class

Reply:
complete